src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static boolean diff_string_check(String[] arr, String toCheckValue)
+    {
+        boolean test = true;
```

Comment:
You might want to name a boolean variable with something that starts with is or has? It will then follow the coding standards.